sql/rikai--0.1.sql

```diff
@@ -26,6 +26,66 @@ AS $$
 	return rikai.__version__.version
 $$ LANGUAGE plpython3u;
 
+CREATE FUNCTION ml.show_info()
+    RETURNS JSON
```

Comment:
It might be better to return `SETOF` as table.

Like two columns, one for name , one for value (version string)
https://www.postgresql.org/docs/current/xfunc-sql.html#XFUNC-SQL-TABLE-FUNCTIONS

Reply:
sg. i changed ml.version to this and show multiple versions.